shells/chrome/src/panel.js

```diff
@@ -42,7 +42,7 @@ var config: Props = {
     // the constructor. Otherwise, inspect the render function.
     var code = `Object.getOwnPropertyDescriptor(window.${vbl}.__proto__.__proto__, 'isMounted') &&
       Object.getOwnPropertyDescriptor(window.${vbl}.__proto__.__proto__, 'isMounted').value ?
-        inspect(window.${vbl}.render) : inspect(window.${vbl}.constructor)`;
+        inspect(window.${vbl}.render) : inspect(window.${vbl}._reactInternalInstance._currentElement.type)`;
```

Comment:
This is a bit gross because we're looking at internals but I don't really have a better alternative. Let's just fix this to work in Fiber: `inspect(typeof window.${vbl}._reactInternalInstance.tag === 'number' ? window.${vbl}._reactInternalInstance.type : window.${vbl}._reactInternalInstance._currentElement.type)`